modules/rhel-images-aws.adoc

```diff
@@ -0,0 +1,29 @@
+[id="rhel-images-aws_{context}"]
+= Listing latest available RHEL images on AWS
+
+// For QE
+// Need a reason why you'd list these images in general; the note with RHEL Workers will limit this to RHEL 7. Also, where else is this used?
+Use the link:https://aws.amazon.com/cli/[AWS Command Line Interface (CLI)] to list the available RHEL images. In the AWS CLI, the following command will generate RHEL 8 Amazon Machine Images (AMI):
```

Comment:
We have attributes for some of our product names. For RHEL, you can reference the attribute in the [common-attributes.adoc](https://github.com/openshift/openshift-docs/blob/master/modules/common-attributes.adoc) file. We typically want to start with the full name, followed by the short name when after it's referenced. So for this sentence, following that guidance, it would look like this:

```
Use the link:https://aws.amazon.com/cli/[AWS Command Line Interface (CLI)] to list the available
{op-system-base-full} images. In the AWS CLI, the following command will generate {op-system-base} 8
Amazon Machine Images (AMI):
```